contracts/v2/EnhancedMainBridgeV2.sol

```diff
@@ -34,7 +34,9 @@ contract EnhancedMainBridgeV2 is EnhancedMainBridgeUpgradeable, OwnableUpgradeab
     uint256 depositCount;
     uint256 requiredSignatures;
 
+    address[] private authorityList;
```

Comment:
authorityList storage variable is declared above authorithyChanging which is declared in the preveious deployed logic contract. So storage slot for authorityList is changed. I think you should move authorityList below authorityChanging.